components/server/ee/src/user/user-service.ts

```diff
@@ -72,9 +73,10 @@ export class UserServiceEE extends UserService {
         }
     }
 
+    // TODO(gpl) Needs to fold into EntitlementService
     async userGetsMoreResources(user: User): Promise<boolean> {
         if (this.config.enablePayment) {
```

Comment:
`if (this.config.enablePayment)` still needed?

Reply:
https://github.com/gitpod-io/gitpod/pull/11831/files/bd28cba0f9d730d84b730dc497832f900b245ef5#r937675184